pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -22,14 +37,43 @@ buildPythonPackage rec {
     sha256 = "35cc8ae9dd5246e2c9861bb796026bbcb9fb083e4d49650f776622171ecdab37";
   };
 
-  nativeBuildInputs = [
-    pkgconfig
-  ];
+  doCheck = false;
 
-  buildInputs = [
-    gtk3 libjpeg libtiff SDL gst-plugins-base libnotify freeglut xorg.libSM
-    which
-  ];
+  nativeBuildInputs = [ pkgconfig which doxygen wxGTK libX11 ];
+
+  buildInputs = with python3Packages; [ libjpeg libtiff SDL
+      gst-plugins-base libnotify freeglut xorg.libSM ncurses
+      requests libpng gstreamer
+      pathlib2
+      (wxGTK.gtk)
+  ]
+    ++ (lib.optional openglSupport pyopengl);
```

Comment:
```suggestion
    ++ lib.optional openglSupport pyopengl;
```

Reply:
According to https://github.com/wxWidgets/Phoenix/blob/wxPython-4.0.6/README.rst#prerequisites and https://github.com/wxWidgets/Phoenix/blob/wxPython-4.0.6/requirements/devel.txt
the following inputs are not needed:
`ncurses`, `libpng`, `pyopengl`

Reply:
@veprbl not sure about ncurses, but think we need the other two: https://github.com/wxWidgets/Phoenix/pull/1291

Reply:
seems like ncurses is required, too:
```
07:08:15 runner ['g++', '-shared', '-Xlinker', '-export-dynamic', 'test.cpp.1.o', '-o/build/wxPython-4.0.6/build/waf/3.7/gtk3/.conf_check_fbce68721686fe9fb8fb624442f27f19/testbuild/testprog.cpython-37m-x86_64-linux-gnu.so', '-Wl,-Bstatic', '-Wl,-Bdynamic', '-L/nix/store/0f331h72l3v42szvcqyc9pqg79ghid9m-python3-3.7.3/lib', '-lpython3.7m', '-lpthread', '-ldl', '-lcrypt', '-lncurses', '-lutil', '-lm', '-lpython3.7m', '-lpthread', '-ldl', '-lcrypt', '-lncurses', '-lutil', '-lm']
```